generator/subaru/subaru_global_2022.dbc

```diff
@@ -0,0 +1,14 @@
+CM_ "IMPORT subaru_global_2017.dbc";
+
+BO_ 292 ES_LKAS_2: 8 XXX
```

Comment:
Is `ES_LKAS_State` (whatever else UI msgs) the same?

Reply:
as far as we know, yes